examples/resource_lacework_alert_channel_datadog/main.tf

```diff
@@ -1,7 +1,21 @@
+terraform {
+  required_providers {
+    lacework = {
+      source = "lacework/lacework"
+      version = "0.5.0"
```

Comment:
How can we source the binary built locally? 🤔  -- This is testing what is in the Terraform Registry, right?